README.md

```diff
@@ -316,17 +337,23 @@ Monitor on the specific model. Clean the cached objects if target are changed.
 
 ### :foreign_key
 
-This option is needed only for caching assoication and need not to set if [`expire_by`](#expire_by) is set to monitor association. Used for backtracing the cache key from cached objects. For examle, if `user` has_many `posts`, and cached the `posts` by user.id. When a post is changed, it needs to know which column to use (in this example, `user_id`) to clean the cache at user.
+- Is needed only for caching assoication
 
-  - Default value is `:id`
+- Does not need to be set if [`expire_by`](#expire_by) is set to monitor association.
 
-  - Will be automatically determined if [`expire_by`](#expire_by) is symbol.
+- Is used for backtracing the cache key from cached objects. For example, it is used if `user` has_many `posts`, and `posts` is cached by user.id. If the post is changed, the column it is going to target must be specified so that the post can clean the cache at user (In this example mentioned, the column was `user_id`).
+
+- Has the default value `:id`.
+
+- Will be automatically determined if [`expire_by`](#expire_by) is symbol
 
 ### :primary_key
 
-This option is needed to know which attribute should be passed to the parameter when you are using instance cacher. For example, if a query, named `email_valid?`, uses `user.email` as parameter, and you call it from instance: `user.cacher.email_valid?`. You need to tell it to pass `user.email` instead of `user.id` as the argument.
+primary_key:
```

Comment:
It seems redundant.
![image](https://user-images.githubusercontent.com/4011729/42312728-ac8637d0-8073-11e8-861c-8c0cf3fc1e65.png)

Reply:
I forgot to remove that. Noted.